src/web/src/asset_POST.ecpp

```diff
@@ -110,13 +111,13 @@ bool database_ready;
         catch (const std::runtime_error &e) {
             log_error ("%s", e.what());
             std::string msg = "Error during configuration sending of asset change notification. Consult system log.";
-            http_die("internal-error", msg.c_str());
+            http_die ("internal-error", TRANSLATE_ME (msg.c_str()));
```

Comment:
I think string should be marked where it's declared, so it would be line 113:
std::string msg = "Error during configuration sending of asset change notification. Consult system log.";

Reply:
well, I prefer to do it same way as other http_dies to avoid forgetting it when change is needed

Reply:
I'd also say it should be marked at it's declaration, otherwise it would be impossible to get the actual string for translations without running any smart variable assignment in the code.

Reply:
If we do it simply like this, I can't imagine gathering the strings. We should do it for all the strings at the declaration of such string, or of const strings, etc to be able to "grep" the source for marked strings and then provide them for translation.